test/com/google/enterprise/adaptor/sharepoint/SharePointAdaptorTest.java

```diff
@@ -1679,6 +1680,155 @@ public void testGetDocContentSubSiteUniquePermissions() throws Exception {
   }
 
   @Test
+  public void testGetDocContentSubSiteUniquePermissionsInvalidUser()
```

Comment:
Quick question before I dive in here: did you add two tests on purpose to test different things, or did you just copy both methods I mentioned as starting points? (I had mentioned both only because they were the two tests that touched the code in question, not because I thought we needed variations on those two tests that threw exceptions, though perhaps we do.)